Sources/CoursesUxMiamFramework/ui/RecipeCard/CoursesUStandaloneRecipeCard.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  CoursesURecipeCard.swift
+//  MiamIOSFramework
+//
+//  Created by Vincent Kergonna on 26/05/2023.
+//  Copyright © 2023 Miam. All rights reserved.
+//
+
+import SwiftUI
+import MiamIOSFramework
+import miamCore
+import MealzUIModuleIOS
+
+@available(iOS 14, *)
+public struct CoursesUStandaloneRecipeCard: CatalogRecipeCardProtocol {
+    let showYellowBanner: Bool
+    let showingOnCatalogResults: Bool
+    public init(
+        showYellowBanner: Bool = false,
+        showingOnCatalogResults: Bool = false
+    ) {
+        self.showYellowBanner = showYellowBanner
+        self.showingOnCatalogResults = showingOnCatalogResults
+    }
+    public func content(params: CatalogRecipeCardParameters) -> some View {
+        let dimensions = Dimension.sharedInstance
+        let callToActionHeight: CGFloat = 70
+        let pictureHeight = params.recipeCardDimensions.height - callToActionHeight
+        
+        
+        func showTimeAndDifficulty() -> Bool {
+            return params.recipeCardDimensions.height >= 320
+        }
+        
+        return VStack(spacing: 0.0) {
+            HStack(spacing: 0.0) {
+                GeometryReader { reader in
+                    ZStack {
+                        AsyncImage(url: params.recipe.pictureURL) { image in
+                            image
+                                .resizable()
+                                .aspectRatio(contentMode: .fill)
+                                .padding(0)
+                                .frame(width: reader.size.width,  height: 200)
+                                .clipped()
+                        }
+                        .contentShape(Rectangle()) // this fixes gesture detector overflow to other cards
+                        .padding(0)
+                        
+                        VStack(spacing: 0) {
+                            HStack {
+                                if showYellowBanner {
+                                    Image(packageResource: "MealIdeas", ofType: "png")
+                                        .resizable()
+                                        .frame(width: 119, height: 40)
+                                    Spacer()
+                                }
+                            }.padding(dimensions.mPadding)
+                            Spacer()
+                            HStack {
+                                /*if params.recipe.isSponsored{
+                                    if let urlString = params.recipe.relationships?.sponsors?.data.first?.attributes?.logoUrl, let url = URL(string: urlString) {
```

Comment:
should this still be commented?

Reply:
Yes, it's to be the same as android, until we can make a correction